### DISCUSSION TITLE
Pretrained Models --> Commercial use

### DISCUSSION THREAD

**Discussion by amaanansari** [pyannote/pyannote-audio#1175] [General]
Hi,

I'm a student and I really like your pyannote.audio library :D At the moment I'm doing a small hobby project but I don't know yet whether I'll might sell it to other's.

I saw that your Code is published under the MIT license. However, I would like to use the pretrained speaker segmentation model. In case I decide to pursue commercial interests, would it be allowed to download the model from hugging-face and ship it then with the program itself?

Thanks :D

**Comment by hbredin** [comment]
The model itself is a also MIT. See the license tag on  Huggingface.
